### ISSUE TITLE
Suggestion: Use oboInOwl:hasRelatedSynonym rather than skos:relatedMatch

### ISSUE THREAD

**Issue by henrietteharmse** [LUMC-BioSemantics/dcat-extension#1]
Currently this extension uses skos:relatedMatch to indicate a related term. As an example, here is the definition of Guideline:

```
dcatext:Guideline rdf:type owl:Class ;
                  rdfs:subClassOf dcat:Resource ;
                  oboInOwl:hasAlternativeId "http://purl.org/ejp-rd/vocabulary/Guideline"^^xsd:anyURI ;
                  rdfs:comment "This class is used to describe a guideline"@en ;
                  rdfs:label "Guideline"@en ;
                  skos:relatedMatch "http://purl.obolibrary.org/obo/NCIT_C17564"@en .
```

The use of `skos:relatedMatch` here seems to be incorrect in that it is defined as string. A more correct definition seems to be: 

```
dcatext:Guideline rdf:type owl:Class ;
                  rdfs:subClassOf dcat:Resource ;
                  oboInOwl:hasAlternativeId "http://purl.org/ejp-rd/vocabulary/Guideline"^^xsd:anyURI ;
                  rdfs:comment "This class is used to describe a guideline"@en ;
                  rdfs:label "Guideline"@en ;
                  skos:relatedMatch <http://purl.obolibrary.org/obo/NCIT_C17564> .
```

However, you are already using oboInOwl and it therefore seems to make sense to use oboInOwl:hasRelatedSynonym. Besides consistency (using oboInOWL throughout rather than a mix of skos and oboInOwl) there are 2 additional reasons why you may want to prefer oboInOwl:hasRelatedSynonym:

1. skos:relatedMatch uses object properties rather than annotation properties. Because it uses object properties, it forces the use of punning.
2.  oboInOwl:hasRelatedSynonym makes use of annotation properties which seems to be the general direction for biological ontologies.

**Comment by rajaram5** [comment]
Hi @henrietteharmse , Are you referring to annotations of class like `Biobank` and `Patient registry`?

**Comment by henrietteharmse** [comment]
@rajaram5 Sorry for the initial cryptic comment... that is what happens when I press ENTER accidentally!

My unease with `skos:relatedMatch` is that it is a object property rather than an annotation property. Object properties can have reasoning implications whereas annotation properties are aimed at providing additional information without any affect on reasoning. From that perspective using annotation properties rather than object properties when there is no intent of using reasoning.

My further unease with `skos:relatedMatch` (and its many varients) is that the semantics of it is not precise enough to enable reasoning on it, even if reasoning was the intention.

**Comment by rajaram5** [comment]
Thanks for the explanation @henrietteharmse . I see your point with the `skos:relatedMatch` I basically try to link other ontology classes with it this annotation so these URIs are not synonyms. Is there a `relatedResource` or `relatedClass` annotation property in `oboIn`?

**Comment by henrietteharmse** [comment]
The short answer is that how you have dealt with `skos:relatedMatch` is fine.

The longer answer: Because you redefine `skos:relatedMatch` as an annotation property, reasoners will not try to reason on it. This to me is counter intuitive and goes against the SKOS spec. That is why I would be hesitant to do that. But I did a bit of asking around, and to my surprise, this is something that is actually done rather often in bio ontologies. See [this discussion](https://github.com/information-artifact-ontology/ontology-metadata/issues/65) for example.

I will close the ticket. Nothing to do here. But at least I have learned a lot in the process.
